src/cljs/ataru/hakija/hakija_readonly.cljs

```diff
@@ -163,17 +163,30 @@
         (< 0 (count answer-value))
         true))))
 
-(defn- followups [followups content application lang question-group-index]
+(defn- selectable [content application lang]
   [:div
-   (text content application lang question-group-index)
-   (into [:div]
-     (for [followup followups
-           :let [followup-is-visible? (get-in @(subscribe [:state-query [:application :ui]]) [(keyword (:id followup)) :visible?])]
-           :when (if (boolean? followup-is-visible?)
-                   followup-is-visible?
-                   (followup-has-answer? followup application))]
-       [:div
-        [field followup application lang question-group-index]]))])
+   [:div.application__form-field-label (some (:label content) [lang :fi :sv :en])]
+   [:div.application-handling__nested-container
+    (let [values           (-> application
+                               (get-in [:answers (keyword (:id content))])
+                               :value
+                               vector
+                               flatten
+                               set)
+          selected-options (filter #(contains? values (:value %)) (:options content))]
+      (doall
+        (for [option selected-options]
+          ^{:key (:value option)}
+          [:div
+           [:p.application__text-field-paragraph (some (:label option) [lang :fi :sv :en])]
+           (into [:div.application-handling__nested-container]
+                 (for [followup (:followups option)
+                       :let [followup-is-visible? (get-in @(subscribe [:state-query [:application :ui]]) [(keyword (:id followup)) :visible?])]
+                       :when (if (boolean? followup-is-visible?)
+                               followup-is-visible?
+                               (followup-has-answer? followup application))]
+                   [:div
```

Comment:
Tän `div`:n voinee poistaa. Tuolta `field` tulevan `nil` ei muistaakseni pitäis haitata mitään.